routes/common.js

```diff
@@ -2,6 +2,7 @@ var path = require('path');
 var fs = require('fs');
 var lunr = require('lunr');
 var sanitizeHtml = require('sanitize-html');
+var objectAssign = require('object-assign');
```

Comment:
Had to install this package to maintain the compatibility with Node version 0.12.
Otherwise, we could just use Object.assign API.